pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -97,18 +101,28 @@ func (cu containersByEvictUnit) NumEvictUnits() int {
 }
 
 // Newest first.
```

Comment:
is this comment still applicable is the sorting is by attempt? Can old container have a very big attempt number?

Reply:
Attempt number is increased when change is detecated.   https://github.com/kubernetes/kubernetes/blob/d140769e4d26f3a175d8a59782ea63595a9f4899/pkg/kubelet/kuberuntime/kuberuntime_container.go#L147-L151     
https://github.com/kubernetes/kubernetes/blob/d140769e4d26f3a175d8a59782ea63595a9f4899/pkg/kubelet/kuberuntime/kuberuntime_manager.go#L442-L445